packages/forma-36-react-components/src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -162,10 +195,17 @@ export const Autocomplete = <T extends {}>({
       if (listRef.current) {
         scrollToItem(listRef.current, newIndex);
       }
-      dispatch({ type: NAVIGATED_ITEMS, payload: newIndex });
-    } else if (isEnter && hasUserSelection) {
+      dispatch({
+        type: NAVIGATED_ITEMS,
+        payload: newIndex,
+        multiselect: !!multiselect,
+      });
+    } else if (
+      (isEnter && hasUserSelection) ||
+      (isSpace && multiselect && hasUserSelection)
+    ) {
       const selected = items[highlightedItemIndex as number];
-      selectItem(selected);
+      selectItem(selected, highlightedItemIndex || 0);
```

Comment:
Do we still need the `|| 0` since in the function declaration the second argument defaults to 0?